lib/galaxy/datatypes/text.py

```diff
@@ -700,6 +700,57 @@ def _looks_like_is_vitesscejson(self, file_prefix: FilePrefix, load_size: int =
         return False
 
 
+@build_sniff_from_prefix
+class AuspiceJson(Json):
+    """
+    Auspice is a visualization tool for phylogenetic trees and associated data.
+    It uses JSON format to represent the tree structure and metadata.
+    """
+
+    file_ext = "auspicejson"
```

Comment:
```suggestion
    file_ext = "auspice.json"
```
? That way your downloads are handled nicely by whatever is associated with json ?